### PR TITLE
change refs to avoid error

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

interminttne update_tests failing due conflict with services

Fixes # (issue)

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [X] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by thom-at-redhat — APPROVED** [review]

File: pkg/services/repobuilder_test.go [hunk 1]
```diff
@@ -67,19 +67,18 @@ var _ = Describe("RepoBuilder Service Test", func() {
...
 		var fileName = "repo.tar"
 		BeforeEach(func() {
-			var ctx context.Context = context.Background()
 
 			ctrl = gomock.NewController(GinkgoT())
 			mockFilesService = mock_services.NewMockFilesService(ctrl)
-			mockDownloaderService = mock_services.NewMockDownloader(ctrl)
-
-			downloadservice = services.RepoBuilder{
+			defer ctrl.Finish()
```
Comment (de1987) [inline pkg/services/repobuilder_test.go:79]: Is the `ctrl.Finish()` on `AfterEach` still being used?
If it's not should it be removed?

Comment (ldjebran) [reply pkg/services/repobuilder_test.go:79]: Carl.Finish() should be in ArterEach it make no sense to defer it here , the mock expect are not counted in this case

Comment (ldjebran) [reply pkg/services/repobuilder_test.go:79]: defer ctrl.Finish() should be removed

**Review by de1987 — DISMISSED** [review]

**Review by de1987 — DISMISSED** [review]

File: pkg/services/updates_test.go [hunk 2]
```diff
@@ -242,8 +245,12 @@ var _ = Describe("UpdateService Basic functions", func() {
 			var uuid string
 			var device models.Device
 			var update models.UpdateTransaction
-
+			var cfg *config.EdgeConfig
 			BeforeEach(func() {
+				cfg = config.Get()
+				cfg.TemplatesPath = fmt.Sprintf("%v/%v/", templatesPath, "../templates")
+
+				fmt.Printf("\n cfg.TemplatesPath %v \n", cfg.TemplatesPath)
```
Comment (omaciel) [inline pkg/services/updates_test.go:253]: Should this be logged instead?

File: pkg/services/updates_test.go [hunk 3]
```diff
@@ -242,8 +245,12 @@ var _ = Describe("UpdateService Basic functions", func() {
 			var uuid string
 			var device models.Device
 			var update models.UpdateTransaction
-
+			var cfg *config.EdgeConfig
 			BeforeEach(func() {
+				cfg = config.Get()
+				cfg.TemplatesPath = fmt.Sprintf("%v/%v/", templatesPath, "../templates")
```
Comment (omaciel) [inline pkg/services/updates_test.go:251]: templatesPath already has "../templates" in it, no?

**Review by omaciel — COMMENTED** [review]
Left a couple of questions but it lgtm overall

**Review by omaciel — APPROVED** [review]
/lgtm

**Comment by omaciel** [comment]
Merging with 1 approval. Asking for forgiveness later, if needed :)

**Pull request merged**